README.md

```diff
@@ -54,61 +79,104 @@ CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
 CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api
 ```
 
-Note that if you are connecting as a host, the login should be prefixed with `host/`. For example: `host/myhost.example.com`,
-not just `myhost.example.com`.
+_NOTE:_ Defining environment variables is dependent on where the application is running. For example, if the Java app is running in Docker, you would set them in the `docker-compose.yml`.
+If the app is running locally, you can configure them in the IDE.
+
+In order to set CONJUR_AUTHN_LOGIN and CONJUR_AUTHN_API_KEY, you will need to add the host to Conjur with the proper privileges in policy. 
+
+For this _specific_ example, we will define our Java application as a host and grant permissions over some variable. This variable will later hold a secret and we will 
+show how to retrieve this secret using the Conjur Java API once a connection is established. If you have a different use case, you can use the below policy as the basis for 
+another desired outcome. If a policy already exists for your host, you can skip the next set of steps.
+
+1. Make sure Conjur is running in the background and open the Conjur CLI.
+    
+    _NOTE:_ If using Docker: `docker-compose exec <NAME_OF_CLI_CONTAINER> bash`
 
-In addition, to run the integration tests you will need to load a Conjur policy. Save this file as `root.yml`:
+2. Copy the following policy, substitute the name of the policy, hostname, and permissions accordingly, and save it as a `.yml` file. 
 
-```yaml
+```xml
 - !policy
-  id: test
+  id: <POLICY_ID>
   body:
+    - !host
+      id: <NAME_OF_HOST>
+
     - !variable
-      id: testVariable
+      id: <NAME_OF_VARIABLE>
+
+    - !permit
+      role: !host <NAME_OF_HOST>
+      privileges: [read, execute]
+      resource: !variable <NAME_OF_VARIABLE>
 ```
 
-To load the policy, use the CLI command `conjur policy load root root.yml`
+3. Load the policy in the Conjur CLI like so: `conjur policy load <NAME_OF_POLICY>.yml`
 
-### SSL Certificates
+4. Once the policy is loaded (for the first time), an ID and API key will be returned for the host. It should look something like:
+```bash
+Loaded policy 'root'
+{
+  "created_roles": {
+    "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>": {
+      "id": "cucumber:host:<POLICY_ID>/<NAME_OF_HOST>",
+      "api_key": "3scdjaz3aq90vn26avdv11s2mf2h2wnwfzw3cbgbja1jspzyc1rdzwqx"
+    }
+  },
+  "version": 1
+}
+```
+
+5. Set the ID and API key as the environment variables CONJUR_AUTHN_LOGIN and CONJUR_AUTHN_API_KEY respectively. 
```

Comment:
`Set the ID and the API key returned by the previous command as the environment variables`